app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java

```diff
@@ -52,14 +62,20 @@
     public static final Logger LOGGER = Logger.getLogger("LocationFragmentViewModel");
     public static final String FLOORS_AVAILABLE = "floorsAvailable";
 
+
+    private PolylineOptions displayedPolylineOption;
+    private Polyline currentlyDisplayedLine;
+    private HashMap<String, List<WalkingPoint>> walkingPointsMap = new HashMap<>();
```

Comment:
Can we change these to more descriptive names? With our changes for the POI, we will be having a looot of walking point lists in this class.